zerver/views/realm.py

```diff
@@ -92,8 +94,9 @@ def update_realm(request, user_profile, name=REQ(validator=check_string, default
                                      message_content_edit_limit_seconds)
         data['allow_message_editing'] = allow_message_editing
         data['message_content_edit_limit_seconds'] = message_content_edit_limit_seconds
-    # Realm.notifications_stream is not a boolean, Text or integer field, and thus doesn't fit
-    # into the do_set_realm_property framework.
+
+    # Realm.notifications_stream, Realm.signup_notifications_stream are not boolean,
```

Comment:
-> Realm.notifications_stream and Realm.signup_notifications_stream are not boolean, Text, or integer fields, and thus don't fit into ..

There's also an extra newline added